tester/src/test/java/com/graphicsfuzz/tester/GeneratorUnitTest.java

```diff
@@ -195,8 +198,9 @@ public void testWrap() throws Exception {
     testTransformationMultiVersions(() -> new AddWrappingConditionalTransformation(),
         TransformationProbabilities.onlyWrap(),
         "wrap.frag",
-        Arrays.asList("colorgrid_modulo.json", "prefix_sum.json"),
-        Arrays.asList("colorgrid_modulo.json", "prefix_sum.json"));
+        Arrays.asList("bubblesort_flag.json"),
+        Arrays.asList("bubblesort_flag.json"));
+    // Reason for blacklisting^: slow.
```

Comment:
Could you file an issue that summarizes the problems with blacklisting shaders?

Reply:
This could probably be treated as the same issue as "the tests are slow". Issue #284